packages/teams/src/remove-team-from-items.ts

```diff
@@ -0,0 +1,45 @@
+import { UserSession } from "@esri/arcgis-rest-auth";
+import { IUpdateItemResponse, updateItem } from "@esri/arcgis-rest-portal";
+import { cloneObject, IModel, without } from "@esri/hub-common";
+import { removeTeam } from "./remove-team";
+
+/**
+ * Removes a Team from N hub models.
+ * First, if passed deleteTeam, it will delete the team
```

Comment:
Would this be better as a separate call rather than trying to add complexity to this method?

Reply:
Although this may have been written as a single method in Ember, I agree with Andrew - keep this simpler, and remove the team via the `removeTeam` fn vs here